pkg/reconciler/kubernetes/tektonresult/tektonresult.go

```diff
@@ -298,6 +302,59 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tr *v1alpha1.TektonResul
 	return nil
 }
 
+// This function exists to resolve conflict between PreReconciler and Reconciler Installerset.
```

Comment:
What is the conflict?

Reply:
Both have db manifest. But we don't need this manifest.